flink-cdc-base/src/test/java/com/ververica/cdc/connectors/base/MySqlChangeEventSourceExampleTest.java

```diff
@@ -102,6 +127,155 @@ public void testConsumingAllEvents() throws Exception {
         env.execute("Print MySQL Snapshot + Binlog");
     }
 
+    @Test
+    @Ignore("Test ignored because it won't stop and is used for manual test")
+    public void testConsumingAllEvents() throws Exception {
+        final DataType dataType =
+                DataTypes.ROW(
+                        DataTypes.FIELD("id", DataTypes.BIGINT()),
+                        DataTypes.FIELD("name", DataTypes.STRING()),
+                        DataTypes.FIELD("description", DataTypes.STRING()),
+                        DataTypes.FIELD("weight", DataTypes.FLOAT()));
+
+        inventoryDatabase.createAndInitialize();
+        final String tableId = inventoryDatabase.getDatabaseName() + ".products";
+        JdbcIncrementalSource<RowData> mySqlChangeEventSource =
+                new MySqlSourceBuilder()
+                        .hostname(MYSQL_CONTAINER.getHost())
+                        .port(MYSQL_CONTAINER.getDatabasePort())
+                        .databaseList(inventoryDatabase.getDatabaseName())
+                        .tableList(tableId)
+                        .username(inventoryDatabase.getUsername())
+                        .password(inventoryDatabase.getPassword())
+                        .serverId("5401-5404")
+                        .deserializer(buildRowDataDebeziumDeserializeSchema(dataType))
+                        .includeSchemaChanges(true) // output the schema changes as well
+                        .splitSize(2)
+                        .build();
+
+        StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
+
+        // enable checkpoint
+        env.enableCheckpointing(3000);
+        // set the source parallelism to 4
+        CloseableIterator<RowData> iterator =
+                env.fromSource(
+                                mySqlChangeEventSource,
+                                WatermarkStrategy.noWatermarks(),
+                                "MySqlParallelSource")
+                        .setParallelism(4)
+                        .executeAndCollect(); // collect record
+
+        String[] snapshotExpectedRecords =
+                new String[] {
+                    "+I[101, scooter, Small 2-wheel scooter, 3.14]",
+                    "+I[102, car battery, 12V car battery, 8.1]",
+                    "+I[103, 12-pack drill bits, 12-pack of drill bits with sizes ranging from #40 to #3, 0.8]",
+                    "+I[104, hammer, 12oz carpenter's hammer, 0.75]",
+                    "+I[105, hammer, 14oz carpenter's hammer, 0.875]",
+                    "+I[106, hammer, 16oz carpenter's hammer, 1.0]",
+                    "+I[107, rocks, box of assorted rocks, 5.3]",
+                    "+I[108, jacket, water resistent black wind breaker, 0.1]",
+                    "+I[109, spare tire, 24 inch spare tire, 22.2]",
+                    "+I[110, spare tire, 28 inch spare tire, 26.2]"
```

Comment:
I don't think this line needs to be added. It will be insered in method makeBinlogEvents